processing/src/main/java/io/druid/query/metadata/SegmentMetadataQueryQueryToolChest.java

```diff
@@ -222,6 +222,18 @@ public SegmentAnalysis apply(@Nullable SegmentAnalysis input)
           }
         };
       }
+
+      @Override
+      public Function<SegmentAnalysis, SegmentAnalysis> prepareForResultLevelCache()
+      {
+        return prepareForCache();
+      }
```

Comment:
can we make these default implementations in CacheStrategy interface itself , so that they are not repeated everywhere ?